packages/components/src/color-palette/test/utils.ts

```diff
@@ -38,4 +39,25 @@ describe( 'ColorPalette: Utils', () => {
 			expect( showTransparentBackground( '#f5f5f524' ) ).toBe( false ); // 0.14 alpha.
 		} );
 	} );
+
+	describe( 'normalizeColorValue', () => {
+		test( 'should return the value as is if the color value is not a CSS variable', () => {
+			const element = document.createElement( 'div' );
+			expect( normalizeColorValue( '#ff0000', element ) ).toBe(
+				'#ff0000'
+			);
+		} );
+		test( 'should return the background color computed from a element if the color value is a CSS variable', () => {
+			const element = document.createElement( 'div' );
+			const { ownerDocument } = element;
+			const { defaultView } = ownerDocument;
+			// @ts-ignore
+			defaultView.getComputedStyle = () => ( {
+				backgroundColor: '#ff0000',
+			} );
```

Comment:
Not ideal, of course, but better than nothing. I'm not sure if @mirka has any idea to who we could get `getComputedStyle` to work our tests (afterall, it seems to work as expected in [`Theme`'s tests](https://github.com/WordPress/gutenberg/blob/e7f07a9487a571bb3df957da51cf5c13f5d85400/packages/components/src/theme/test/index.tsx)).

Reply:
Or even @jsnajdr , given his recent work on `getComputedStyle`

Reply:
JSDOM has a very limited support for CSS, and it doesn't expand CSS variables to their real values. That means that `normalizeValue` is not possible to unit test with Jest and JSDOM. I would consider if I want to keep the unit test at all.

A less harsh way to "fake" the CSS value is to set the style on the tested element itself:
```js
element.style = { backgroundColor: '#ff0000' };
```
Then `getComputedStyle( element )` will find this style and will return it.

By the way, the `normalizeValue` function would be better off if its second param was just the element, not a ref:
```ts
normalizeColorValue( value: string | undefined, element: HTMLElement | undefined )
```

Reply:
Thank you, Jarda!

> A less harsh way to "fake" the CSS value is to set the style on the tested element itself:

Good point. @t-hamano , would you mind trying out this approach?

> By the way, the normalizeValue function would be better off if its second param was just the element, not a ref:

Agreed! Although this change should have been applied in a [previous commit](https://github.com/WordPress/gutenberg/pull/47373/commits/011e099bfaaa8c684ca7394753a7a5ceb5293a6e)